src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -21,7 +21,7 @@ public sealed partial class AnalyzerConfig
         // Matches EditorConfig section header such as "[*.{js,py}]", see https://editorconfig.org for details
         private static readonly Regex s_sectionMatcher = new Regex(@"^\s*\[(([^#;]|\\#|\\;)+)\]\s*([#;].*)?$", RegexOptions.Compiled);
         // Matches EditorConfig property such as "indent_style = space", see https://editorconfig.org for details
-        private static readonly Regex s_propertyMatcher = new Regex(@"^\s*([\w\.\-_]+)\s*[=:]\s*(.*?)\s*([#;].*)?$", RegexOptions.Compiled);
+        private static readonly Regex s_propertyMatcher = new Regex(@"^\s*([\w\.\-_]+)\s*[=:]\s*(.*?)\s*$", RegexOptions.Compiled);
```

Comment:
> [=:]

Why are we considering both `=` and `:` to separate the key and the value? I don't think `key : value` should be allowed.

Reply:
➡️ I have no idea, but that seems beyond the scope of this change.